### ISSUE TITLE
Contradictory information regarding invariant vs current culture use in string.Equals

### ISSUE THREAD

**Issue by IanKemp** [dotnet/docs#25440]
According to https://docs.microsoft.com/en-us/dotnet/csharp/how-to/compare-strings#default-ordinal-comparisons:

> By default, the most common operations:
>
> * String.Equals
> * String.Equality and String.Inequality, that is, equality operators == and !=, respectively
>
> perform a case-sensitive ordinal comparison **and, if necessary, use the current culture**.

However the help page for `string.Equals` **explicitly states that the invariant culture is used** https://docs.microsoft.com/en-us/dotnet/api/system.string.equals?view=netcore-3.1#System_String_Equals_System_String_System_String_:

> ## Remarks
>
> This method performs an ordinal (case-sensitive and **culture-insensitive**) comparison.

These two pieces of documentation are contradictory. Please can the incorrect one be fixed.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 11ecfc18-477b-f792-4136-2feb46bbc5d3
* Version Independent ID: 82f53ae8-a9d4-dea9-5acd-6d9202a40882
* Content: [How to compare strings - C# Guide](https://docs.microsoft.com/en-us/dotnet/csharp/how-to/compare-strings)
* Content Source: [docs/csharp/how-to/compare-strings.md](https://github.com/dotnet/docs/blob/main/docs/csharp/how-to/compare-strings.md)
* Product: **dotnet-csharp**
* Technology: **csharp-fundamentals**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

**Comment by gewarren** [comment]
You can double check with @safern, but I believe the API docs are correct, because further down in the C# article, it implies that `String.Equals` and `==`/`=!` don't use the current culture:

> The test for equality with String.Equals and the == and != operators **differs from string comparison using the String.CompareTo and Compare(String, String) methods**. While the tests for equality perform a case-sensitive ordinal comparison, **the comparison methods perform a case-sensitive, culture-sensitive comparison using the current culture**. Because the default comparison methods often perform different types of comparisons, we recommend that you always make the intent of your code clear by calling an overload that explicitly specifies the type of comparison to perform.

**Comment by safern** [comment]
Correct, `string.Equals`, `==`/`=!` are Ordinal comparisons and are not culture aware. They are case sensitive by default.

The only case where `string.Equals` would be culture aware is if it is specified via the override that takes a `StringComparison` option. 

cc: @tarekgh

**Comment by BillWagner** [comment]
Adding this to the current sprint, and adding the "up for grabs" label, as the fix is noted in the comment above.

**Comment by strobelt** [comment]
I created a PR for this [here](https://github.com/dotnet/docs/pull/26449). Let me know if this is good enough! Any changes are welcome too.
